app/components/app_gillick_card_component.html.erb

```diff
@@ -8,13 +8,14 @@
           summary_list.with_row do |row|
             row.with_key { "Assessment" }
             row.with_value do
-              "Assessed by #{@patient_session.gillick_competence_assessor.full_name}" +
-              tag.br +
-              "#{@consent.recorded_at || @consent.created_at}.to_fs(:nhsuk_date)"
+              safe_join([
+                "Assessed by #{@assessment.assessor.full_name}",
+                (@assessment.recorded_at || @assessment.created_at).to_fs(:nhsuk_date),
+              ], tag.br)
```

Comment:
Thanks, was there an injection here without the `safe_join`? 🤔 

I did this change recently as part of the Rufo formatting work

Reply:
Well this bit definitely had a bug in it:

```
"#{@consent.recorded_at || @consent.created_at}.to_fs(:nhsuk_date)"
```

but I wouldn't worry about this component because I think it's going to go away very soon (it's been designed out).